src/remotestorage.js

```diff
@@ -172,20 +172,31 @@ RemoteStorage.prototype = {
   },
 
   /**
-   * TODO: document
+   * Initiate the OAuth authorization flow.
```

Comment:
This should contain a description of why/when/in what context one might want to use the function.

Reply:
Not sure what to put there. Any suggestions?

Reply:
You just implemented it, so you know best about when to use it.

(The way I understand it, this is not usually something one wants to use in an app, but only in a custom storage backend. But I'm not 100% sure.)

Reply:
This is still unanswered.

Reply:
I added a note that this function is not called by an app directly but rather from a custom storage backend implementation.